### PR TITLE
[15.0][IMP] fieldservice_sale_reference: add sale_reference in fsm.order report

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Add sale_reference in reports:
imagen

### CODE REVIEW

File: fieldservice_sale_reference/reports/fsm_order_template.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template
+        id="report_fsm_order_document_sale_reference"
+        inherit_id="fieldservice.report_fsm_order_document"
+    >
+        <xpath expr="//t[@t-if='doc.template_id']" position="after">
+            <span> - <span t-field="doc.sale_reference" /></span>
```
Comment (dalonsod) [inline fieldservice_sale_reference/reports/fsm_order_template.xml]: Empty data protection (not tested):
```suggestion
            <t t-if="doc.sale_reference">
                <span> - <span t-field="doc.sale_reference" /></span>
            </t>
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment.
And add yourself to CONTRIBUTORS

**Comment by ChristianSantamaria** [comment]
Done and I am the only contributor of this addon

**Comment by dalonsod** [comment]
OK, I'll test it ASAP

**Review by dalonsod — APPROVED** [review]
Locally tested 👍 

@ChristianSantamaria you can merge it 😀

**Comment by ChristianSantamaria** [comment]
Let goo!

**Pull request merged**